src/plugins/dashboard/public/application/utils/get_nav_actions.tsx

```diff
@@ -99,7 +99,7 @@ export const getNavActions = (
       stateContainer.transitions.set('title', newTitle);
       stateContainer.transitions.set('description', newDescription);
       stateContainer.transitions.set('timeRestore', newTimeRestore);
```

Comment:
not specifically related to this PR but in types we can add to the DashboardAppStateTransitions

`setDashboard: (state: DashboardAppState) => (dashboard: Partial<DashboardAppState>) => DashboardAppState`

which should hopefully improve the run time of the application, since we listen to state changes everywhere and so each one of these cause the app state to be updated and then trigger any listeners to app state updates. So I believe in this instance it's updating the state 3 times and triggering listeners to react 3 times.

vs just adding the above and doing:
```tsx
stateContainer.transitions.setDashboard({
  title: newTitle,
  description: newDescription,
  timeRestore: newTimeRestore
});
````

Reply:
yea agree, was originally planning to clean that part after i test out the functionality